packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -24,116 +20,89 @@ in another terminal session by running this command from this directory:
- * @param {options.runKbn}  method to start kibana
- * }
+ * @param {string[]} configPaths   Array of paths to configs
+ * @param {boolean}  bail          Whether to exit test run at the first failure
+ * @param {Log}      log           Optional logger
  */
-export async function runTests(
-  configPaths,
-  options = {
-    runEs: runElasticsearch,
-    runKbn: runKibanaServer,
-  }
-) {
+export async function runTests(configPaths, { bail, log }) {
   if (!configPaths || configPaths.length === 0) {
-    throw new Error('runTests requires configPaths to contain at least one item');
+    log.error(`runTests requires configPaths to contain at least one item`);
+    process.exit(1);
   }
 
   for (const configPath of configPaths) {
     try {
-      await runSingleConfig(resolve(KIBANA_ROOT, configPath), options);
+      await runSingleConfig(resolve(process.cwd(), configPath), { bail, log });
     } catch (err) {
-      fatalErrorHandler(err);
+      fatalErrorHandler(err, { log });
     }
   }
 }
 
-
 /**
  * Start only servers using single config
- * @param {configPath}      path to a config file
- * @param {options}         optional
- * @param {options.runEs}   method to start elasticsearch
- * @param {options.runKbn}  method to start kibana
+ * @param {string}  configPath   Path to a config file
  */
-export async function startServers(
-  configPath,
-  options = {
-    runEs: runElasticsearch,
-    runKbn: runKibanaServer,
-  }
-) {
+export async function startServers(configPath, { log }) {
   if (!configPath) {
-    throw new Error('startServers requires configPath');
+    log.error(`startServers requires configPath`);
+    process.exit(1);
   }
 
-  const configOption = getopts(process.argv.slice(2)).config;
-
-  configPath = await resolve(KIBANA_ROOT, configOption || configPath);
+  configPath = resolve(process.cwd(), configPath);
 
   try {
-    await withTmpDir(async tmpDir => {
-      await withProcRunner(async procs => {
-        const config = await readConfigFile(log, configPath);
-        const { runEs, runKbn } = options;
-
-        const es = await runEs({ tmpDir, config });
-        await runKbn({ procs, config });
-
-        // wait for 5 seconds of silence before logging the success message
-        // so that it doesn't get buried
-        await Rx.Observable.fromEvent(log, 'data')
-          .switchMap(() => Rx.Observable.timer(5000))
-          .first()
-          .toPromise();
-
-        log.info(SUCCESS_MESSAGE);
-        await procs.waitForAllToStop();
-        await es.cleanup();
-      });
+    await withProcRunner(log, async procs => {
+      const config = await readConfigFile(log, configPath);
+
+      const es = await runElasticsearch({ config, log });
+      await runKibanaServer({ procs, config, log });
+
+      // wait for 5 seconds of silence before logging the success message
+      // so that it doesn't get buried
+      await Rx.Observable.fromEvent(log, 'data')
```

Comment:
This should be updated to match https://github.com/elastic/kibana/blob/master/x-pack/dev-tools/functional_tests/tasks.js#L174-L178

Reply:
Ah, thank you!